build/build-image/cross/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN go get golang.org/x/tools/cmd/cover \
             golang.org/x/tools/cmd/goimports
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v3.0.17; \
+RUN export ETCD_VERSION=v3.1.10; \
```

Comment:
need to update build/build-image/cross/VERSION as well.

Reply:
Increment the last number? Like v1.8.3-2 -> v1.8.3-3 ?

Reply:
yes